frontend/src/hooks/entities/auth-hooks.ts

```diff
@@ -68,6 +70,7 @@ export const useLogout = (
     },
     onSuccess: async () => {
       queryClient.removeQueries([CURRENT_USER_KEY]);
+      postDisconnectionSignal("logout");
```

Comment:
Does the same tab receive the message it sends ?

Reply:
Yes, the same tab receive the message sent